.github/workflows/pr-e2e-testing.yaml

```
@@ -67,6 +73,8 @@ jobs:
       - name: Run E2E Tests
         if: ${{ steps.filter.outputs.notdocs == 'true' }}
         run:  make run-e2e-test VERBOSE=true # use DEBUG_VM_CONSOLE=1 to see the VM console output
+        env:  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)